src/handlebars/supported_platforms.handlebars

```diff
@@ -14,7 +14,7 @@
     </div>
     <div class="margin-bottom">
 
-        <h3 id="openjdk8_platforms">OpenJDK 8</h3>
+        <h3 id="openjdk8_platforms">AdoptOpenJDK 8</h3>
         <p>Where available, OpenJDK 8 binaries are supported on the minimum operating system levels shown in the following tables:</p>
```

Comment:
maybe "binaries based on OpenJDK" instead of "OpenJDK binaries"

Reply:
Fixed thank you.